x-pack/plugins/fleet/public/components/agent_enrollment_flyout/hooks.tsx

```diff
@@ -66,4 +70,14 @@ export function useIsK8sPolicy(agentPolicy?: AgentPolicy) {
   return { isK8s };
 }
 
-const isK8sPackage = (pkg: PackagePolicy) => K8S_PACKAGES.has(pkg.package?.name as string);
```

Comment:
this is a bit problematic since `pkg.package?` can return `undefined` and will throw when used in `.has()`.